truss/truss_handle.py

```diff
@@ -1075,7 +1076,22 @@ def _wait_for_docker_build(container) -> None:
     ),
 )
 def _wait_for_model_server(url: str) -> Response:
-    return requests.get(url)
+    logger.info(f"Waiting for model server to be ready at {url}")
```

Comment:
This change is required by integration test so we can properly wait for the docker server to be up before hit it. @squidarth Please help double check it wont break anything.

Reply:
This has a bigger blast radius, since this functionality is used in a bunch of places. It makes sense though, with a few pieces of feedback:
* Please use [tenacity](https://tenacity.readthedocs.io/en/latest/) instead of doing our own retry mechanism, and have a limit on the number of retries
* Remove the log statements